### PR TITLE
Sort features in Readme by area instead of by type

### PR DESCRIPTION
**Pull request by fregante**
It's easier to keep track of what features applies where.

Also I'm rewriting the descriptions to follow the style "Refined Github <verb>" instead of "Refined GitHub allows you to <verb>" which often results in awkward phrasing.

### CODE REVIEW

**Comment by fregante** [comment]
This PR is huge and I don't expect anyone to actually review it. But if you do, I'll apply those changes in a separate PR.

Things to check:

- Readability (best seen in the rendered view)
- Possible duplicates
- Possible name, description, screenshot mismatches

Refer to the new styleguide https://github.com/sindresorhus/refined-github/blob/2e4a24d3957c205c7bc49525c69e1fa08593d2ca/readme.md#L100

File: readme.md [hunk 1]
```diff
@@ -94,158 +94,229 @@ GitHub Enterprise is also supported. More info in the options.
...
+- <!-- link-to-file-in-file-history --> [Adds links to the file itself in a file’s commit list.](https://user-images.githubusercontent.com/22439276/57195061-b88ddf00-6f6b-11e9-8ad9-13225d09266d.png)
+- <!-- extend-diff-expander --> [Widens the `Expand diff` button to be clickable across the screen.](https://user-images.githubusercontent.com/6978877/34470024-eee4f43e-ef20-11e7-9036-65094bd58960.PNG)
+- <!-- remove-diff-signs --> [Hides diff signs](https://user-images.githubusercontent.com/1402241/54807718-149cec80-4cb9-11e9-869c-e265863211e3.png) (since diffs are color coded already.)
+- <!-- limit-commit-title-length --> [Limits the commit title fields to 72 characters.](https://user-images.githubusercontent.com/37769974/60379478-106b3280-9a51-11e9-88b9-0e3607f214cd.gif)
+
+<!-- Refer to style guide above. Keep this message between sections. -->
+
+### Profiles
+
+- <!--  --> [Shows the followers you have in common.](https://user-images.githubusercontent.com/2906365/42009293-b1503f62-7a57-11e8-88f5-9c2fb3651a14.png)
```
Comment (fregante) [inline readme.md]: For those wondering, these comments are just so features can be easily found and removed in the readme when they're dropped/renamed/updated.

```suggestion
- <!-- show-followers-you-know --> [Shows the followers you have in common.](https://user-images.githubusercontent.com/2906365/42009293-b1503f62-7a57-11e8-88f5-9c2fb3651a14.png)
```

File: readme.md [hunk 2]
```diff
@@ -76,10 +76,10 @@ GitHub Enterprise is also supported. More info in the options.
 <table>
 	<tr>
 		<th width="50%">
-			Clickable references to issues and pull requests
+			Linkifies issue/PR references and URLs in code
 		</th>
 		<th width="50%">
-			Links to an issue's closing commit or pull request
+			Add link to an issue’s closing commit or pull request
```
Comment (fregante) [inline readme.md]:
```suggestion
			Adds link to an issue’s closing commit or pull request
```

**Comment by busches** [comment]
Looks solid. 😀

**Comment by FloEdelmann** [comment]
Looks like GitHub doesn't parse Markdown following HTML (comments) in the same line:

Screenshot_2019-07-29_21-19-41

Notice that the link/code in "New projects can still be found …" still works, as there is no comment.

**Pull request merged**